release/nightly_tests/dataset/gpu_batch_inference.py

```diff
@@ -1,13 +1,13 @@
 from typing import Dict
 import click
 import time
-import json
 import os
 
 import numpy as np
 import torch
 from torchvision.models import resnet50, ResNet50_Weights
 
+from benchmark import Benchmark
```

Comment:
note, moved this file and related compute YAMLS to `nightly_tests/dataset/` directory, in order to make use of the `Benchmark` class. the import does not work across directories since they are not located in a package